core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -82,6 +79,50 @@ def start_navigator(self) -> None:
     def start_serial(self, device: str) -> None:
         self.start_mavlink_manager(Endpoint("Master", self.settings.app_name, "serial", device, 115200, protected=True))
 
+    @staticmethod
+    def get_current_arch() -> str:
+        """Maps paltform.machine() outputs to docker architectures"""
+        arch_map = {"x86_64": "amd64", "aarch64": "arm", "armv7l": "arm"}  # TODO: differentiate arm64v8/ arm32v7
```

Comment:
if it does not contains x86 is arm, you don't need to have all necessary cases for it, ardupilot only supports x86 and arm.

Reply:
Just to make sure, you're saying to remove aarch64 (ARM64), correct?

Reply:
Removed aarch64.